trace/src/main.rs

```diff
@@ -164,8 +164,8 @@ fn main() -> anyhow::Result<()> {
                 analyze_trace(reader);
             }
         }
-        Command::Draw => todo!(),
-        Command::Replay => todo!(),
+        Command::Draw => unimplemented!(),
+        Command::Replay => unimplemented!(),
```

Comment:
I am late to the party but I think this hunk demonstrates that we don't really need this at all.